src/e

```diff
@@ -279,6 +279,33 @@ program
     process.exit(0);
   });
 
+program
+  .command('shell')
+  .description(
+    'Launch a shell environment populated with build-tools environment variables and context',
+  )
+  .action(() => {
+    depot.ensure();
+
+    if (process.platform !== 'linux' && process.platform !== 'darwin') {
+      fatal('e shell is not supported on non-unix platforms');
+    }
+
+    if (!process.env.SHELL) {
+      fatal('Could not detect shell to launch');
+    }
+
+    console.info(`Launching build-tools shell with ${color.cmd(process.env.SHELL)}`);
+    const { status } = depot.spawnSync(evmConfig.current(), process.env.SHELL, [], {
+      stdio: 'inherit',
+      env: {
+        ...process.env,
+        SHELL_CONTEXT: ` ${evmConfig.currentName()}`,
```

Comment:
Can this be replaced with the Unicode escape code, `\ue62e`? And/or, should the unicode character be included at all? Seems specific to your setup.

Reply:
Most people have this unicode character in their terminals, any of the nerdfonts / powerline fonts have it

Reply:
Ok. Still would prefer it use the escape code.